### PR TITLE
Increase required PHP version to 8.1

### PR DESCRIPTION
**Pull request by leofeyer**
@ausi Would you prefer `^3.4 || ^4.0 || ^5.0 || ^6.0 || ^7.0` over `">=3.4"` as version constraint for the Symfony packages? Can we maybe drop some old versions?

And can we make PHP 8.1 the minimum version?

### CODE REVIEW

**Comment by ausi** [comment]
> Would you prefer `^3.4 || ^4.0 || ^5.0 || ^6.0 || ^7.0` over `">=3.4"` as version constraint for the Symfony packages? Can we maybe drop some old versions?

I changed it to `^5.4 || ^6.4 || ^7.0` in #22

**Comment by ausi** [comment]
> And can we make PHP 8.1 the minimum version?

Yes, please ☺️

**Comment by ausi** [comment]
I reverted the README in 659187865906a6204848f61d030fb5808a310b34 as I use [semantic line breaks](https://sembr.org/) there.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,34 +11,6 @@ on:
         - cron: 0 13 * * MON,TUE
 
 jobs:
-    coverage:
-        name: Coverage
```
Comment (ausi) [inline .github/workflows/ci.yml]: Did you delete the coverage job on purpose? Or is this only temporarily?

**Comment by ausi** [comment]
Rest looks good to me. Thank you! ❤️

File: .github/workflows/ci.yml (hunk 1 above)
Comment (leofeyer) [reply .github/workflows/ci.yml]: <del>On purpose. We do not even use Codecov anymore, do we?</del>

Re-added in f205d058bb1316cb4c1756330003bc9d60dc644e.

**Comment by leofeyer** [comment]
> I reverted the README in [6591878](https://github.com/contao/monorepo-tools/commit/659187865906a6204848f61d030fb5808a310b34) as I use [semantic line breaks](https://sembr.org/) there.

I didn‘t know that. Sorry. 🙈

File: composer.json [hunk 2]
```diff
@@ -44,25 +43,21 @@
...
-        ],
-        "unit-tests": [
-            "vendor/bin/phpunit --colors=always"
-        ]
-    },
-    "config": {
-        "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
-        }
+        "ecs": "@php vendor/bin/ecs check --fix --ansi",
```
Comment (ausi) [inline composer.json:58]: I’d prefer using the same name as we have in contao/contao WDYT?

```suggestion
        "cs-fixer": "@php vendor/bin/ecs check --fix --ansi",
```

Comment (leofeyer) [reply composer.json:58]: We will soon have `ecs` there, too, because we use the tool name as key everywhere else.

**Review by ausi — APPROVED** [review]
Thank you very much! ❤️

**Pull request merged**